sys/libjade/Cargo.toml

```diff
@@ -12,6 +12,8 @@ log = "0.4"
 [build-dependencies]
 libc = { version = "0.2", default-features = false }
 fs_extra = "1.2"
+cc = { version = "1.0", features = ["parallel"] }
+libcrux_platform = { path = "../platform" }
```

Comment:
Please fix this to the version number of the platform crate (`=0.0.1`)

Reply:
Done!